app/javascript/packs/application.js

```diff
@@ -25,11 +23,8 @@ global.Spinner = Spinner
 global.Clipboard = clipboard
 global.Cookies = Cookies
 
```

Comment:
Not any current line related, but I see that you put the toolkit.js in the webpack managed stuff, but it doesn't seem to be imported anywhere, which means it probably won't ever be ran.  Not sure if they do anything special on top of what bootstrap is doing.

Reply:
Nice thanks, probably nothing vital but I'm sure this callout saved a future headache